### ISSUE TITLE
Need for a small aligned dataset

### ISSUE THREAD

**Issue by PhyloStar** [PhyloStar/PyBayes#29]
Need a aligned dataset for testing the phylogeny.  Something like Bulgarian hand-aligned will suffice. May be @LinguList can provide one. The idea is to test out the software on linguistic datasets to see how fast it can infer phylogenies.

**Comment by LinguList** [comment]
You would want to have a good account on the real phylogeny underlying it, right? So I'd not recommend the Bulgarian data, as it is not instructive, but maybe the Germanic subset from the phonetic alignment benchmark? Please have a look at http://alignments.lingpy.org (all is also for download). there are 110 concepts, and we could easily get 20 languages, where we have glottocodes and expert phylogenies.

**Comment by PhyloStar** [comment]
Okay. Kewl. This is useful. My understanding of the Germanic dataset is that the alignments are within a single cognate set. Is that right @LinguList ?

**Comment by LinguList** [comment]
yes! we just need a mapping of major doculects to glottolog, and all is
there (I reckon, about 20 of the varieties), but this will again be
phonetic data.

**Comment by PhyloStar** [comment]
Kewl. My aim is to test the software on Phonetic data and see if it can infer the right tree from alignments. I would treat Gaps as any other character. The usual phylogenetic programs treat gaps as missing character.

**Comment by LinguList** [comment]
This is dangerous, as gaps behave differently. But you should use different scores for gaps, depending on position (extensively described in my thesis).

**Comment by PhyloStar** [comment]
Okay. That would be assuming that all transitions in and out of gaps have same score. That will reduce the number of parameters to estimate.

Position can be dealt with a Gamma rates distribution.

**Comment by PhyloStar** [comment]
@LinguList Need to reduce the alphabet size. The current alphabet size is 222.  The program works but is comparatively slower. We will need a alphabet that is smaller. I added a file based on the data from website for Germanic doculects: `data/German_multistate.phy`

**Comment by LinguList** [comment]
Yep, could be done in various ways. E.g., our [clpa](http://github.com/glottolog/clpa) could be used to break down the fine-grained distinctions, or a rough procedure removing diacritics. This will need time, as it requires looking into the data and making bold decisions. A shortcut would be to convert to things like asjp using lingpy...
